packages/office-ui-fabric-react/src/components/Button/Button.tsx

```diff
@@ -19,7 +25,7 @@ export interface IButtonState {
 
 export class Button extends BaseComponent<IButtonProps, IButtonState> implements IButton {
```

Comment:
What do we need to do here to ensure that console warnings are shown when `<Button>` is used?